usaspending_api/api_contracts/contracts/references/Agency.md

```diff
@@ -25,50 +25,19 @@ This endpoint returns a high-level overview of a specific government agency, giv
 
 # Data Structures
 
-## PageMetadataObject (object)
-+ count (required, number)
-+ page (required, number)
-+ has_next_page (required, boolean)
-+ has_previous_page (required, boolean)
-+ next (required, nullable , string)
-+ current (required, string)
-+ previous (required, nullable, string)
-
 ## AgencyOverview (object)
-+ active_fy: `2017` (required, string)
++ `active_fy` (required, string)
     The fiscal year that the data in the response reflects.
-+ active_fq: `2` (required, string)
++ `active_fq` (required, string)
     The latest quarter of the `active_fy` that the data in the response reflects.
-+ agency_name: Department of the Treasury (required, string)
-+ mission: `Maintain a strong economy and create economic and job opportunities by promoting the conditions that enable economic growth and stability at home and abroad, strengthen national security by combating threats and protecting the integrity of the financial system, and manage the U.S. Government's finances and resources effectively.` (required, string)
++ `agency_name` (required, string)
++ `mission` (required, string)
     The agency's mission statement.
-+ icon_filename: `DOT.jpg` (required, string)
++ `icon_filename` (required, string)
     The file name of the agency's logo. If no logo is available, this will be an empty string. Images can be found at `https://www.usaspending.gov/graphics/agency/[file]`.
-+ website: `https://www.treasury.gov` (required, string)
-+ budget_authority_amount: 1899160740172.16 (required, string)
-+ current_total_budget_authority_amount: 8361447130497.72 (required, string)
-+ obligated_amount: 524341511584.82 (required, string)
-+ outlay_amount: 523146830716.62 (required, string)
-
-## FinancialBalance (object)
-+ budget_authority_amount: 1899160740172.16 (required, string)
-+ obligated_amount: 524341511584.82 (required, string)
-+ outlay_amount: 523146830716.62 (required, string)
-
-
-## MajorObjectClass (object)
-+ major_object_class_code: 30 (required, string)
-+ major_object_class_name: Acquisition of assets (required, string)
-+ obligated_amount: 216067467.29 (required, string)
-
-## MinorObjectClass (object)
-+ object_class_code: 310 (required, string)
-+ object_class_name: Equipment (required, string)
-+ obligated_amount: 209728763.65 (required, string)
-
-## FederalAccount (object)
-+ account_title (required, string)
-+ account_number (required, string)
-+ id (required, string)
-    The USAspending.gov unique identifier for the federal account. You will need to use this ID when making API requests for details about specific federal accounts.
-+ obligated_amount (required, string)
++ `website` (required, string)
++ `congressional_justification_url` (required, nullable, string)
++ `budget_authority_amount` (required, string)
++ `current_total_budget_authority_amount` (required, string)
++ `obligated_amount` (required, string)
++ `outlay_amount` (required, string)
```

Comment:
I thought we weren't doing backticks...

Reply:
@tony-sappe

Reply:
You have to do backticks if the value or name has special characters (such as underscores) in it.  I'm pretty sure we've "standardized" on backticks for names and string values.

Reply:
Although I guess we've done away with values...

Reply:
Have we? I feel like that was mostly me advocating and the team never fully getting around to having a meeting on it.

Reply:
@BrianZito What gave you the impression we were not doing backticks? After we learned they were necessary we have been increasing the usage of them.

Reply:
I recall a PR where I was asked (I think by you) to remove them, because they were based on a misunderstanding of the syntax by previous developers, and that moving forward we were going to take them off of future blueprints. It would take a bit to dig up the exact PR.

Reply:
Odd, I cannot recall any event I would have said that. The only possibility I can think of is if someone placed backticks around a number value or object name.